using-list-methods

```diff
@@ -0,0 +1,9 @@
+# In python we gt many options to combine elements of list into single string.
+
+arr = ['H','e','l','l','o',' ','W','o','r','l','d']   # initiating the list with elements
+
+print("".join(arr))  # using join method with double quote
+print(''.join(arr))  # using join method with single quote
+print(*arr,sep='')   # using * operator with sep attribute
```

Comment:
This method has also been [implemented](https://github.com/Goku-kun/1000-ways-to-print-hello-world-in-python/blob/master/using-iterable-inpacking.py) under the iterable unpacking.

Reply:
Sure, after going through the [code](https://github.com/Goku-kun/1000-ways-to-print-hello-world-in-python/blob/master/using-iterable-inpacking.py) I've seen that the implementation is between 2 elements rather than 10 elements. Also, the ```sep``` method is not used in the above code but I've used the method.

Reply:
Okay. That's interesting. How about we only keep the `sep` implementation then? I checked and there is not a specific `sep` based implementation. We'll remove all the other methods and solely keep this one and rename the file to 'using-print-sep.py'. What do you think about that?

Reply:
That's okay for me. Will it count to my contribution? Also the file_name is probably correct & makes sense to me but if you're eradicating other's contributions, what'll happen to their code?

Reply:
Hello. How am I eradicating anything at all? I think you're misunderstanding something here. Kindly elaborate upon what you mean by "eradicating other's contributions". Sir, I believe you've been misled about how Github works. Even if I do make changes to your contribution, it still **counts as your contribution**. That's the essence of distributed version control. However, I hold the rights to how anyone can contribute since I'm the owner of this repository and that's true for any open source project.

Reply:
As you told me, **Eradicating the contribution** means the code which other contributors have provided is being changed by my code, as per my understanding. As per you, you want to only keep the ```sep``` implementation rather than both right? If that's not the case, it's fine for me. Please clarify more if needed. Thanks.

Reply:
Uh. No. I'm not eradicating anyone's code. LOL. In fact, **to preserve other's contributions** is precisely why I pointed out that parts of your code have already been implemented. And I also said that the only implementation from your contribution I'll be able to add to the repository will be the 'sep' one because no one other than you has added it before.

Reply:
Okay, thanks. I'm ok with it.